nebula-exchange/src/test/scala/com/vesoft/nebula/exchange/utils/NebulaUtilsSuite.scala

```diff
@@ -0,0 +1,165 @@
+                            "col2",
+                            "col3",
+                            "col4",
+                            "col5",
+                            "col6",
+                            "col7",
+                            "col8",
+                            "col9",
+                            "col10",
+                            "col11",
+                            "col12")
+    val sourceFields = List("col1",
+                            "col2",
+                            "col3",
+                            "col4",
+                            "col5",
+                            "col6",
+                            "col7",
+                            "col8",
+                            "col9",
+                            "col10",
+                            "col11",
+                            "col12")
+    val label = "person"
+    val sourceConfig = new TagConfigEntry(label,
+                                          null,
+                                          null,
+                                          sourceFields,
+                                          nebulaFields,
+                                          null,
+                                          Some(KeyPolicy.UUID),
+                                          1,
+                                          1,
+                                          Some(""))
+
+    val space   = "testSST1"
+    val address = new ListBuffer[HostAndPort]()
+    address.append(HostAndPort.fromParts("127.0.0.1", 9559))
+    val metaProvider = new MetaProvider(address.toList)
+
+    val map: Map[String, Int] =
+      NebulaUtils.getDataSourceFieldType(sourceConfig, space, metaProvider)
+    assert(map("col1") == PropertyType.STRING)
+    assert(map("col2") == PropertyType.FIXED_STRING)
+    assert(map("col3") == PropertyType.INT8)
+    assert(map("col4") == PropertyType.INT16)
+    assert(map("col5") == PropertyType.INT32)
+    assert(map("col6") == PropertyType.INT64)
+    assert(map("col7") == PropertyType.DATE)
+    assert(map("col8") == PropertyType.DATETIME)
+    assert(map("col9") == PropertyType.TIMESTAMP)
+    assert(map("col10") == PropertyType.BOOL)
+    assert(map("col11") == PropertyType.DOUBLE)
+    assert(map("col12") == PropertyType.FLOAT)
+  }
+
+  @Test
+  def getPartitionId(): Unit = {
+    val storageClient = new StorageClient("127.0.0.1", 9559)
+    storageClient.connect()
+    for (i <- 1 to 12) {
+      val vid            = i + ""
+      val partitionId    = NebulaUtils.getPartitionId(vid, 10, VidType.STRING)
+      val scanResultIter = storageClient.scanVertex("testSST1", partitionId, "person")
+      var containVertex  = false
+      while (scanResultIter.hasNext) {
+        val scanResult = scanResultIter.next()
+        val map        = scanResult.getVidVertices
+        for (value <- map.keySet().asScala) {
+          if (value.asString().equals(vid)) {
+            containVertex = true
```

Comment:
if there one is equal, the `containVertex` is true, but not means all value are equal

Reply:
If there's vertex equals vid, which means this vid is in `partitionId` partition, so the computation of `getPatitionId()` is correct.